mails/themes/classic/core/bankwire.html.twig

```diff
@@ -0,0 +1,84 @@
+{% extends '@MailThemes/classic/components/layout.html.twig' %}
+
+{% block content %}
+<tr>
+  <td align="center" class="titleblock">
+    <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
+      <span class="title">{{ 'Hi {firstname} {lastname},'|trans({}, 'EmailsBody', locale)|raw }}</span><br/>
+      <span class="subtitle">{{ 'Thank you for shopping with {shop_name}!'|trans({}, 'EmailsBody', locale)|raw }}</span>
+    </font>
+  </td>
+</tr>
+<tr>
+  <td class="space_footer">&nbsp;</td>
+</tr>
+<tr>
+  <td class="box" style="border:1px solid #D6D4D4;">
+    <table class="table">
+      <tr>
+        <td width="10">&nbsp;</td>
+        <td>
+          <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
+            <p data-html-only="1" style="border-bottom:1px solid #D6D4D4;">
+              {{ 'Order {order_name}'|trans({}, 'EmailsBody', locale)|raw }}&nbsp;-&nbsp;{{ 'Awaiting wire payment'|trans({}, 'EmailsBody', locale)|raw }}
```

Comment:
same here for raw

Reply:
I updated the templates so that `raw` is not used when there are variables (unless it contains html code).